docs/source/conf.py

```diff
@@ -93,6 +93,12 @@
 #
 # html_theme_options = {}
 
+# The name of an image file (relative to this directory) to use as a favicon of
+# the docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
+# pixels large.
+#
```

Comment:
A nit but let's remove the comment as it basically duplicates the documentation.

Reply:
ok I'll remove it.

Reply:
@hvy Removed it.

Reply:
Thanks for the swift action!